spec/services/resync_service_spec.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
+
+RSpec.describe ResyncService do
+  describe ".call" do
+    before do
+      stub_any_publishing_api_publish
+      stub_any_publishing_api_put_content
+      allow(PoliticalEditionIdentifier)
+        .to receive(:new)
+        .and_return(instance_double(PoliticalEditionIdentifier, political?: true))
+    end
+
+    context "when there is no live edition" do
+      let(:document) { create(:document, :with_current_edition) }
+
+      it "synchronises the current edition, but does not publish" do
+        expect(PreviewService).to receive(:call).with(document.current_edition).and_call_original
```

Comment:
Do you need the call_original?

Reply:
Yes, so we can check that the revision has been synced,